webapp/portlet/src/main/webapp/activity-composer-app/components/ExoActivityComposer.vue

```diff
@@ -87,7 +93,7 @@ export default {
   },
   data() {
     return {
-      MESSAGE_MAX_LENGTH: 2000,
+      MESSAGE_MAX_LENGTH: 1300,
```

Comment:
Max length is still 2000. 1300 is for displaying the message "Switch to an article" and change the characters count in red.